File: bddtests/src/main/kotlin/mocking/vision/models/OrderNewPrescriptionRequest.kt
package mocking.vision.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlElement

@XmlAccessorType(XmlAccessType.FIELD)
data class OrderNewPrescriptionRequest(
        @XmlElement(namespace = "urn:vision")
        var patientId: String,

        @XmlElement(namespace = "urn:vision")
        var repeat: List<NewPrescriptionRepeat>,

        @XmlElement(namespace = "urn:vision")
        var message: String
)

File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Vision/Models/Prescriptions/OrderNewPrescriptionRequest.cs
using System.Collections.Generic;
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Vision.Models.Prescriptions
{
    public class OrderNewPrescriptionRequest
    {
        [XmlElement(ElementName = "patientId", Namespace = "urn:vision")]
        public string PatientId { get; set; }

        [XmlElement(ElementName = "repeat", Namespace = "urn:vision")]
        public List<NewPrescriptionRepeat> Repeats { get; set; } = new List<NewPrescriptionRepeat>();

        [XmlElement(ElementName = "message", Namespace = "urn:vision")]
        public string Message { get; set; }
    }
}
